components/Button.vue

```diff
@@ -58,74 +53,204 @@ export default class Button extends Vue {
   // Class of the append wrapper
   @Prop(String) readonly appendClass!: string | undefined
 
-  // Disable interaction if set to true
-  @Prop({ default: false }) readonly isDisabled!: boolean
-
   // Equivalent to `to` of `<NuxtLink/>`
   @Prop({ default: null }) readonly to: object | null | undefined
 
   get rootClassesForPreset(): any {
     switch (this.preset) {
       case 'primary':
         return [
-          'bg-like-green',
-          'text-like-cyan',
+          'bg-like-green', //
+          'text-like-cyan-light',
         ]
 
       case 'secondary':
         return [
-          'bg-like-cyan',
+          'bg-like-cyan-light', //
           'text-like-green',
         ]
 
       case 'tertiary':
         return [
-          'bg-[#EBEBEB]',
-          'text-gray-4a',
+          'bg-shade-gray',
+          {
+            'text-dark-gray': !this.circle,
+            'text-like-green': this.circle,
+          },
```

Comment:
如果是二選一，可以用 Conditional operator
```suggestion
          this.circle ? 'text-like-green' : 'text-dark-gray',
```